shop/.dockerignore

```diff
@@ -9,3 +9,6 @@
 !composer.*
 !yarn.*
 !.stylelint*
+!tests/
+!setup_test
```

Comment:
reason for this might be:

```
./vendor/spryker/application/tests/SprykerTest/Shared/Application/_support/Helper/SetupHelper.php:    const TEST_ENV_SCRIPT = 'setup_test';
./vendor/spryker/application/tests/_support/Module/Infrastructure.php:    const TEST_ENV_SCRIPT = 'setup_test';
```